content/docs/5.1/deploy/operate/scale.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Scale
+description: Learn how to scale out/in TiKV using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 2
+---
+
+The capacity of a TiKV cluster can be increased or decreased without interrupting the online services.
+
+This document describes how to scale the TiKV and PD using TiUP.
+
+To view the current cluster name list, run `tiup cluster list`.
```

Comment:
what is the name list?

Reply:
it's not necessary to list the cluster names here.

Reply:
I mean, what does `name list` means?

Reply:
we can assume that there is only one target cluster to be scaled.

Reply:
this line is a little bit strange.